tests/lib/ocpp/v16/.gitignore

```diff
@@ -0,0 +1 @@
+/*.pdf
```

Comment:
Can be removed?

Reply:
yes, but only because the md file is no longer in that directory.
it was there to avoid markdown to PDF files from accidentally being included in git